package.json

```diff
@@ -8,6 +8,7 @@
   "license": "GPL-3.0",
   "dependencies": {
     "irc": "^0.5.2",
+    "jsonfile": "^3.0.0",
```

Comment:
If we do it that way, we would have to use fs, read a string and then use
JSON.parse(). This module does that for us, leaving us to do the relatively
simple stuff.